### PR TITLE
Adding topics 1 and 2

### PR DESCRIPTION
**Pull request by audstav**
@seanlip @DubeySandeep PTAL

### CODE REVIEW

File: docs/_static/theme_override.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (seanlip) [inline docs/_static/theme_override.css:13]: Drop newline at top.

Include standard copyright notice -- see https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/css/oppia.css for an example.

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    'sphinx.ext.autodoc',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
```
Comment (seanlip) [inline docs/conf.py:51]: Delete unused code if you're replacing it; don't comment it out or check it in.

File: docs/keyconcepts.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+.. _topic:
+
+**Topic:** A topic refers to the subject matter to be taught (e.g., Decimals, Fractions). A topic will consist of a set of stories. 
+
+.. _story:
+
+**Story:** Stories are non-contrived (realistic) scenarios that teach the topic. If the topic is Decimals, then the first story - aiming to introduce the concept - might be a scenario where a customer is in a shop and has to pay an amount in dollars and cents. A story consists of a set of :ref:`explorations <explorations>` or lessons.
+
+.. note:: 
+   Stories within a topic have to be linear, i.e., a student must complete a story before beginning the next story.
```
Comment (seanlip) [inline docs/keyconcepts.rst]: This isn't true. Chapters are linear within a story, but stories can be played in any order.

File: docs/keyconcepts.rst [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+   
+   *Topic hierarchy in Oppia*
+
+.. _subtopic:
+
+**Subtopic:** In Oppia, a subtopic refers to a group of skills within a topic. Subtopics are created to provide students the opportunity to review and practice skills. 
+
+.. note:: 
+   * If a skill is not assigned to a subtopic, students will not be able to practice that skill. 
+   * If a subtopic is not created for a topic, students will not be able to practice the topic.
```
Comment (seanlip) [inline docs/keyconcepts.rst]: Not sure what this sentence means...

File: docs/keyconcepts.rst [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+**Interaction:** Interactions refer to the *answer* that the student gives to the content (question asked), and the resulting *response* (feedback) from the tutor. The type of interaction depends on the type of exploration - a Math exploration has different interaction options from a Programming one. 
+
+This diagram summarizes the relationship between the above concepts:
+
+.. figure:: /images/Explorations.png
+   :alt: Explorations hierarchy
+   :align: center
+
+   *Exploration hierarchy in Oppia*
+
```
Comment (seanlip) [inline docs/keyconcepts.rst]: Drop all the unnecessary newlines here and below. (In general, check "Files Changed" after pushing code to ensure that what you are committing is what you really want to commit -- i.e. do a self-review first.)

**Review by seanlip — CHANGES_REQUESTED** [review]
Hi @audstav, this looks _fantastic_. Thank you very much! The video is great, too :)

I left a few inline comments  -- please take a look. Also, two general comments:

(a) Please add  a .gitignore for the .DS_Store files. We should not be checking them in.
(b) Would it be possible to add a link to a demo ReadTheDocs site showing the changes in a PR, whenever you submit a PR? I think it would be easier to look at that rather than the .rst files.

Thanks!

File: docs/keyconcepts.rst (hunk 3 above)
Comment (aks681) [reply docs/keyconcepts.rst]: Actually, canonical stories (which is the only one we have now) are supposed to be played linearly, right?

File: docs/conf.py (hunk 2 above)
Comment (audstav) [reply docs/conf.py:51]: Can I get more clarification on this? I don't really understand what I need to do. I don't remember changing anything on this section of the file (except for lines 22-24).

File: docs/keyconcepts.rst (hunk 3 above)
Comment (audstav) [reply docs/keyconcepts.rst]: For reference, this is from the **Oppia Structure Project** doc where I got the information from:
 - Users will be shown a linear, ordered set of stories associated with the topic. A story must be completed before beginning the next story.
 - The story page will display the (potentially) nonlinear list of lessons that make up the narrative of the story.
Has this changed?

File: docs/keyconcepts.rst (hunk 4 above)
Comment (audstav) [reply docs/keyconcepts.rst]: Do you mean it's confusing with the way I worded it? Or the information itself is incorrect? (I can reword it if it's a case of it sounding confusing.)
I got this from the **Oppia Structure Project** doc again: 

_Not all skills must be assigned to a subtopic, but this will surface a warning if not the case, as learners will not be able to practice skills not associated with a subtopic. If no subtopics are present for a topic, then learners will not have the ability to practice the topic._

File: docs/conf.py (hunk 2 above)
Comment (seanlip) [reply docs/conf.py:51]: Ah, sorry! On re-reading, this is my mistake, I didn't realize that this line was still part of the instructions. Please ignore this comment.

File: docs/keyconcepts.rst (hunk 3 above)
Comment (seanlip) [reply docs/keyconcepts.rst]: Yes, I think so. I think it's reversed actually. Stories within a topic do not need to be linear. But lessons/chapters within a story are linear.

File: docs/keyconcepts.rst (hunk 4 above)
Comment (seanlip) [reply docs/keyconcepts.rst]: Ah, it is technically correct, but the parallelism with the bullet point above it is a little confusing since the meaning of the second word, "a", changes between the two. I suggest rewording the second bullet point as "If no subtopics are created for a topic" or "If a topic contains no subtopics".

File: docs/keyconcepts.rst (hunk 3 above)
Comment (seanlip) [reply docs/keyconcepts.rst]: @aks681 Canonical stories can be played/accessed in any order. We do have a suggested order (i.e. the order they are displayed on the page), but there is no obligation for learners to follow that order.

**Comment by DubeySandeep** [comment]
@audstav, If you have resolved the review comments can you please reply to them as stated [here](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#address-review-comments-until-all-reviewers-give-lgtm-looks-good-to-me) in the wiki page? (Ignore if it's not done yet!)

File: docs/_static/theme_override.css (hunk 1 above)
Comment (audstav) [reply docs/_static/theme_override.css:13]: Done

File: docs/keyconcepts.rst (hunk 3 above)
Comment (audstav) [reply docs/keyconcepts.rst]: Done

File: docs/keyconcepts.rst (hunk 4 above)
Comment (audstav) [reply docs/keyconcepts.rst]: Done

File: docs/keyconcepts.rst (hunk 5 above)
Comment (audstav) [reply docs/keyconcepts.rst]: Done

**Comment by audstav** [comment]
@seanlip @DubeySandeep I've addressed the requested changes, PTAL

File: docs/_static/theme_override.css [hunk 6]
```diff
@@ -1,3 +1,15 @@
+/*
+  Copyright 2014 The Oppia Authors. All Rights Reserved.
```
Comment (seanlip) [inline docs/_static/theme_override.css]: Let's change this to 2019

File: docs/_static/theme_override.css [hunk 7]
```diff
@@ -6,5 +18,4 @@
 
 .wy-nav-top {
     background: #009688;   
-}
-
+}
```
Comment (seanlip) [inline docs/_static/theme_override.css]: Note that the last _character_ in a file should be a newline. You can check the "Files Changed" tab to see if you've done this correctly -- it will show a red circle if not.

(The original comment was to remove the additional blank line, but this line should still be newline-terminated.)

File: docs/keyconcepts.rst [hunk 8]
```diff
@@ -20,10 +20,7 @@ Key Terms used in Oppia
 
 .. _story:
 
-**Story:** Stories are non-contrived (realistic) scenarios that teach the topic. If the topic is Decimals, then the first story - aiming to introduce the concept - might be a scenario where a customer is in a shop and has to pay an amount in dollars and cents. A story consists of a set of :ref:`explorations <explorations>` or lessons.
-
-.. note:: 
-   Stories within a topic have to be linear, i.e., a student must complete a story before beginning the next story.
+**Story:** Stories are non-contrived (realistic) scenarios that teach the topic. If the topic is Decimals, then the first story - aiming to introduce the concept - might be a scenario where a customer is in a shop and has to pay an amount in dollars and cents. A story consists of a linearly ordered set of :ref:`explorations <explorations>` or lessons, i.e., within a story, a student must complete the first lesson before moving on to the second.
```
Comment (seanlip) [inline docs/keyconcepts.rst]: One nitpicky question -- should the hyphens, here and elsewhere, be em-dashes instead?

**Review by seanlip — COMMENTED** [review]
Thanks @audstav! Just a couple more comments and then it's ready to go.

File: docs/conf.py [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'Oppia'
+copyright = u'2019, Oppia'
+author = u'Audrey Tavares'
+
+# The short X.Y version
+version = u''
```
Comment (DubeySandeep) [inline docs/conf.py]: Make it `0.1`.

File: docs/conf.py [hunk 10]
```diff
@@ -0,0 +1,183 @@
...
+# The full version, including alpha/beta/rc tags
+release = u'3.0'
```
Comment (DubeySandeep) [inline docs/conf.py]: Make this `0.1.0`

File: docs/index.rst [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+Oppia is an open source learning platform with the goal of providing engaging content to learners. Anyone can create and share interactive lessons (explorations) with the aim of recreating a one-on-one tutoring scenario.    
+
+This interactivity is what makes Oppia unique. Many of the educational tools on the Internet provide a unidirectional flow of information which makes it hard to create engaging, feedback-rich experiences for learners. However, Oppia comes with several built-in interactions and responses that make it possible to personalize feedback to a learner's answer. Over time, explorations can be improved as more learners engage with the content. 
+
+Oppia's mission is to provide high-quality, accessible learning experiences to everyone around the world.
+
+To begin, we suggest getting  familiar with the :ref:`key terms used in Oppia <keyconcepts>` first.
+
+
+:ref:`Lesson Creators - start here<create>`
```
Comment (DubeySandeep) [inline docs/index.rst]: Should we add space before `<` here and elsewhere similar to line 14? (or let's change line 14.)

File: docs/conf.py [hunk 12]
```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+#
```
Comment (DubeySandeep) [inline docs/conf.py:2]: @seanlip, should we also add the copyright notice here?

**Review by DubeySandeep — APPROVED** [review]
@audstav, This looks great, I've left a few simple comments PTAL! (LGTM, approved!)

File: docs/index.rst (hunk 11 above)
Comment (seanlip) [reply docs/index.rst]: I think any convention is fine, as long as it's consistent. Probably the one in line 14 is more readable?

File: docs/conf.py (hunk 12 above)
Comment (seanlip) [reply docs/conf.py:2]: No, I don't think it's needed here, since this is a config file that's mostly not written by us.

File: docs/_static/theme_override.css (hunk 6 above)
Comment (audstav) [reply docs/_static/theme_override.css]: Done

File: docs/_static/theme_override.css (hunk 7 above)
Comment (audstav) [reply docs/_static/theme_override.css]: Thanks for the explanation, I hope I've got it right. Done.

File: docs/keyconcepts.rst (hunk 8 above)
Comment (audstav) [reply docs/keyconcepts.rst]: Good point! I have changed where necessary. Done

File: docs/conf.py (hunk 9 above)
Comment (audstav) [reply docs/conf.py]: Done

File: docs/conf.py (hunk 10 above)
Comment (audstav) [reply docs/conf.py]: Done

File: docs/index.rst (hunk 11 above)
Comment (audstav) [reply docs/index.rst]: I have added a space now in all files for consistency. Done

**Comment by audstav** [comment]
@DubeySandeep @seanlip I've updated the PR, PTAL.

File: docs/lesson/create.rst [hunk 13]
```diff
@@ -205,7 +204,7 @@ Add Responses
    
    When a learner answers incorrectly, there is an option to get the learner to 'try again' or loop back and answer the question again. It isn't always a good idea to do this. Think about whether you are able to group the possible 'wrong' answers in such a way that you can provide detailed feedback to the learner. 
 
-   If you cannot do this, i.e., if the possible 'wrong' answers have come in through a text-field (so you have no way of knowing what the learner might type), or if there are lots of wrong answers to a question - it is better to send the learner to a separate branch to address their misconceptions. Looping them back to the question in this case might cause frustration as they will end up trying to guess what you want them to say.
+   If you cannot do this, i.e., if the possible 'wrong' answers have come in through a text-field (so you have no way of knowing what the learner might type), or if there are lots of wrong answers to a question—it is better to send the learner to a separate branch to address their misconceptions. Looping them back to the question in this case might cause frustration as they will end up trying to guess what you want them to say.
```
Comment (seanlip) [inline docs/lesson/create.rst:207]: One question about the em dashes -- should there be single spaces on each side of them? (Throughout.)

**Review by seanlip — APPROVED** [review]
All good! Just one comment about em dashes; but this is otherwise ready for merge.

File: docs/lesson/create.rst (hunk 13 above)
Comment (audstav) [reply docs/lesson/create.rst:207]: This is a style choice, most style guides recommend no spaces before and after the em dash (Associated Press style guide is the exception).
